app/services/search_product.rb

```diff
@@ -2,6 +2,6 @@
 
 class SearchProduct
   def call(phrase)
-     Product.search_by_name(phrase)
+    Product.search_by_name(phrase)
```

Comment:
Jak najbardziej możnaby usunąć ten service, ale wtedy testowanie metody `search_by_name` przeniosło by się do kontrolera, czy gdzie miałbym ją testować? Czy jest ok trzymać taki jednolinijkowy serwis tylko po to, żeby oddzielić testowanie?